src/schema/devcards.ts

```diff
@@ -224,7 +224,7 @@ export const resolvers: IResolvers<any, Context> = {
       ).toString();
       const url = new URL(
         `/devcards/v2/${devCard.userId}.png`,
-        process.env.OG_URL,
```

Comment:
I vote for not removing it completely yet. We may revisit this solution later on and it's not like one more env variable would cause a catastrophic tech debt :)

Reply:
They are also easy to remove. We can keep it until devcards is finished but just make sure to add a task to remove it then if not needed. FYI @nensidosari